po/ja.po

```diff
@@ -18065,30 +18089,42 @@ msgid ""
 "whichever arrives first. Since the dog takes 50ms, it wins against the cat "
 "that take 500ms seconds."
 msgstr ""
+"この例では、猫と犬のレースを扱っています。`first_animal_to_finish_race`は両方"
+"のチャネルをリッスンし、先に到着した方を選びます。犬は到着まで50msかかるの"
+"で、100msかかる猫に勝ちます。"
 
 #: src/async/control-flow/select.md:67
 msgid ""
 "You can use `oneshot` channels in this example as the channels are supposed "
 "to receive only one `send`."
 msgstr ""
+"この例では`oneshot`チャネルを使うこともできます。なぜなら、チャネルは一回きり"
+"の`send`を受け取ることになっているからです。"
 
 #: src/async/control-flow/select.md:70
 msgid ""
 "Try adding a deadline to the race, demonstrating selecting different sorts "
 "of futures."
 msgstr ""
+"レースに制限時間を追加することによって、違う種類のfutureをselectすることを実"
+"演してみてください。"
 
 #: src/async/control-flow/select.md:73
 msgid ""
 "Note that `select!` drops unmatched branches, which cancels their futures. "
 "It is easiest to use when every execution of `select!` creates new futures."
 msgstr ""
+"`select!`はマッチしなかった枝をドロップすることに注意してください。これは、そ"
+"うした枝のfutureがキャンセルされることにつながります。`select!`を毎回実行する"
+"際に新たなfutureが作成されるときに、`select!`を使うのが最も簡単です。"
 
 #: src/async/control-flow/select.md:76
 msgid ""
 "An alternative is to pass `&mut future` instead of the future itself, but "
 "this can lead to issues, further discussed in the pinning slide."
 msgstr ""
+"Futureそのものでなく、`&mut future`を渡すという代替案もあります。しかし、これ"
+"は問題につながることもあります。このことは固定スライドで詳細に議論します。"
```

Comment:
hmm, if I'm not mistaken "the pinning slide" refers to https://google.github.io/comprehensive-rust/async/pitfalls/pin.html, right?
I think the term "pin" should not be translated, in that case.

So for the last sentence, I would put it like below
```suggestion
"は問題につながることもあります。このことはPinに関するスライドで詳細に議論します。"
```